scraper/cazy_webscraper.py

```diff
@@ -498,7 +498,7 @@ def parse_family(family_url, family_name, cazy_home, logger):
 
     # check url formating
     try:
-        re.match(rf"{cazy_home}/(\D\D|\D\D\D)(\d+|\d_\d+)_all.html", first_pagination_url).group()
+        re.match(rf"{cazy_home}/(\D\D|\D\D\D)(\d+|\d+_\d+)_all.html", first_pagination_url).group()
```

Comment:
Regular expressions allow you to specify the number of repeats. For example `\D{2,3}` is equivalent to (and easier to maintain/debug/read) than `\D\D|\D\D\D`.

I'll address this in a commit.